api/openapi-spec/swagger.json

```diff
@@ -17658,6 +17658,11 @@
           "kind": "DeleteOptions",
           "version": "v2beta2"
         },
+        {
+          "group": "autoscaling.karmada.io",
+          "kind": "DeleteOptions",
```

Comment:
Where did this come from?

Reply:
I add status subresource to the federatedHPA api and use /update-swagger.sh to update. I think it's because we add status subresource.